metaci/build/templates/build/build_list.html

```diff
@@ -1,7 +1,5 @@
 {% extends 'layout_full.html' %}
 
-{% block layout_parent_link %}<a href="/builds">Builds</a>{% endblock %}
```

Comment:
This link is redundant. "Builds" is available (and more prominent) in main navbar